runtime/android/core_internal/src/org/xwalk/core/internal/XWalkViewInternal.java

```diff
@@ -863,6 +865,18 @@ public Uri getRemoteDebuggingUrl() {
     }
 
     /**
+     * Create a new InputConnection for and InputMethod to interact with the view.
+     * The default implementation returns the InputConnection created by ContentView
+     * @param outAttrs Fill in with attribute information about the connection
+     * @return the new InputConnection
+     * @since 6.0
+     */
+    @XWalkAPI
+    public InputConnection onCreateInputConnection(EditorInfo outAttrs) {
```

Comment:
I do not see this API is in the embedding API design document, anything I missed?